packages/core/src/common/path.ts

```diff
@@ -49,14 +46,12 @@ export class Path {
         this.isAbsolute = firstIndex === 0;
         this.base = lastIndex === -1 ? raw : raw.substr(lastIndex + 1);
         this.isRoot = this.isAbsolute && firstIndex === lastIndex && (!this.base || Path.isDrive(this.base));
-        this.root = this.computeRoot();
-
         const extIndex = this.base.lastIndexOf('.');
         this.name = extIndex === -1 ? this.base : this.base.substr(0, extIndex);
         this.ext = extIndex === -1 ? '' : this.base.substr(extIndex);
     }
 
-    protected computeRoot(): Path | undefined {
```

Comment:
@hexa00 Could you point out the code where it breaks.

Reply:
I used to have a console.log(`${JSON.stringify(path)}`) and that broke.... I since removed it but I still think it's a good idea to have that just as a accessor

Reply:
you can do `console.log(path)`, `root` accessor will create all parents on each call

Reply:
so you mean with console.log(path) it will still blow up since it will call the accessors?
It's not the case with stringify at least